lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -368,6 +441,11 @@ public function removeFromGroup( \User $performer, \User $userToChange, $groups
 		} finally {
 			$userToChange->invalidateCache(); // it calls $this->invalidateCache( $userToChange );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)

[project-id]: wikia:app
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app